lib/upgradable-multisig/multisig/utils.ak

```diff
@@ -0,0 +1,62 @@
+use aiken/list
+use aiken/transaction.{Input, Output}
+use aiken/transaction/credential.{Credential}
+use aiken/transaction/value.{PolicyId, Value}
+use upgradable_multisig/types.{AssetClass, MultisigDatum, PubKeyHash}
+
+pub fn get_input_by_token(inputs: List<Input>, policy_id: PolicyId) {
+  list.find(
+    inputs,
+    fn(input) { list.has(value.policies(input.output.value), policy_id) },
```

Comment:
We should also check the token name.

Reply:
Removed this function  altogether as I'm checking it in the and block of the validator.